tests/test_discoverer.py

```diff
@@ -73,3 +74,31 @@ def test_discover_removed_file():
     discoverer_test = discoverer_run_once()
     discoverer_test.start()
     assert int(max_db_id()) == 1
+
+
+# Trigger manual rescan
+def test_manual_rescan():
+    # Setup
+    shutil.copy2(file_path + origin_file, file_path + destination_file)
+    process_handle = subprocess.Popen(
+        "python -m flumes.discoverer -d " + file_path + " -i " + db_path, shell=True
```

Comment:
```suggestion
        "python -m flumes.discoverer -d " + file_path + " -i " + sqlite_command, shell=True
```

Reply:
Also you can assign this string `"python -m flumes.discoverer -d " + file_path + " -i " + sqlite_command` to a variable at the beginning of the test instead of having it hardcoded inside Popen.